.gitlab-ci.yml

```diff
@@ -514,9 +513,11 @@ agent_deb-x64-a7:
   <<: *agent_build_common_deb
 
 agent_deb-arm-a6:
-  extends: .agent_build_common_arm
+  stage: package_build
+  needs: ["run_dep_check_lock"]
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deb_arm_x64:$DATADOG_AGENT_ARMBUILDIMAGES
```

Comment:
The image name is `deb_arm64` instead of `deb_arm_x64`. Also `$DATADOG_AGENT_ARMBUILDIMAGES` should have changed since this version comes from a different repo.

Reply:
About the image name, I'm quite surprised. I used the same name as the agent one: https://github.com/DataDog/datadog-agent/blob/39c487df5ea4cd6c36444a388dda24c9bd92f03a/.gitlab-ci.yml#L396

But it's true that the image name on the image repository is `deb_arm64 `.

I'll change the `DATADOG_AGENT_ARMBUILDIMAGES` env variable.